client/agent-opentracing/src/main/java/org/hawkular/apm/agent/opentracing/OpenTracingManager.java

```diff
@@ -375,6 +406,60 @@ public static void reset() {
     }
 
     /**
+     * This method determines if the supplied path should be monitored.
+     *
+     * @param path The path
+     * @return Whether the path is valid for monitoring
+     */
+    public boolean includePath(String path) {
+        // Determine if the path is NOT hawkular-apm related and
+        // the final part of the path is NOT a filename (with extension)
+        return !path.startsWith("/hawkular/apm") && (path.lastIndexOf('.') <= path.lastIndexOf('/'));
+    }
+
+    /**
+     * This method retrieves a variable associated with the
+     * current trace.
+     *
+     * @return The variable value, or null if not found
+     */
+    public String getVariableAsString(String name) {
```

Comment:
No, as the API only enables tags to be set for string, boolean and number types, and I had problems trying to cast within the byteman rules.